include/mbedtls/ecdsa.h

```diff
@@ -255,6 +286,33 @@ int mbedtls_ecdsa_write_signature_det( mbedtls_ecdsa_context *ctx,
 #endif /* MBEDTLS_DEPRECATED_REMOVED */
 #endif /* MBEDTLS_ECDSA_DETERMINISTIC */
 
+/**
+ * \brief           Convert a signature from numbers to ASN.1
```

Comment:
Suggestion: `Convert an ECDSA signature from number pair format to ASN.1.`

Reply:
Done